docs/mainpage.md

```diff
@@ -1,19 +1,22 @@
-# AirshipBindings Portable Class Library
+# AirshipBindings .NET Standard Library and Portable Class Library
 
-The AirshipBindings portable client library exposes a common subset
-of functionality between the iOS and Android SDKs. The PCL can be used within
+The AirshipBindings .NET standard library and portable client library expose a common subset
```

Comment:
Suggestion: The AirshipBindings .NET standard ~library~ and portable client ~library~ libraries expose